### PR TITLE
Update helm chart params with extraLabels field

### PR DESCRIPTION
**Pull request by ezodude**
## Description

Changes proposed in this pull request:

- Adds the new `extraLabels` to the helm chart param documentation.

### CODE REVIEW

**Review by mszostok — COMMENTED** [review]
Please also update contribution guide as it was also changed on https://github.com/kubeshop/botkube/pull/661

File: content/contribute/_index.md [hunk 1]
```diff
@@ -32,26 +32,51 @@ Before you proceed, make sure you have installed BotKube Slack/Mattermost/Teams
...
-2. Deploy newly created image in your cluster.
+2. Deploy the newly created image in your cluster:
 
    ```sh
    helm install botkube --namespace botkube --create-namespace \
    --set communications.slack.enabled=true \
    --set communications.slack.channel=<SLACK_CHANNEL_NAME> \
    --set communications.slack.token=<SLACK_API_TOKEN_FOR_THE_BOT> \
-   --set settings.clusterName=<CLUSTER_NAME> \
+   --set settings.clustername=<CLUSTER_NAME> \
```
Comment (mszostok) [inline content/contribute/_index.md:77]:
```suggestion
   --set settings.clusterName=<CLUSTER_NAME> \
```

I also fixed that on the `botkube` repo: https://github.com/kubeshop/botkube/pull/665

File: content/contribute/_index.md [hunk 2]
```diff
@@ -32,26 +32,51 @@ Before you proceed, make sure you have installed BotKube Slack/Mattermost/Teams
...
+      > **Note**
+      > This command takes some time to run as it builds the images for multiple architectures.
 
-1. This will build BotKube and create a new container image tagged as `ghcr.io/kubeshop/botkube:v9.99.9-dev`
    ```sh
-   make build
    make container-image
    docker tag ghcr.io/kubeshop/botkube:v9.99.9-dev-amd64 <your_account>/botkube:v9.99.9-dev
    docker push <your_account>/botkube:v9.99.9-dev
    ```
```
Comment (mszostok) [inline content/contribute/_index.md:67]: please add a padding to align with a note (I also fixed that on the `botkube` repo: https://github.com/kubeshop/botkube/pull/665)

**Review by mszostok — APPROVED** [review]
LGTM 👍 

but please fix two minor comments before merging. Thanks!

File: content/contribute/_index.md (hunk 2 above)
Comment (ezodude) [reply content/contribute/_index.md:67]: I'll have a look. I think GoLand is removing the padding.

The content was copied directly from [the PR](https://github.com/kubeshop/botkube/pull/665)

File: content/contribute/_index.md (hunk 1 above)
Comment (ezodude) [reply content/contribute/_index.md:79]: Resolving as `clusterName` is already `clustername`.

Comment (mszostok) [reply content/contribute/_index.md:79]: hm.. but it should be `clusterName` not `clustername`

**Pull request merged**